packages/logger/test/custom-logger.test.ts

```diff
@@ -0,0 +1,204 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { Logger, setLogLevel } from '../src/logger';
+import { setUserLogHandler } from '../index';
+
+describe(`Custom log handler`, () => {
+  const client1 = new Logger('@firebase/test-logger');
+  const client2 = new Logger('@firebase/other-logger');
+  let result: any;
+  let spies: any = {};
+
+  describe('Callback log level set to INFO (default)', () => {
+    before(() => {
+      setUserLogHandler((callbackParams) => {
+        result = callbackParams;
+      });
+    });
+    
+    beforeEach(() => {
+      result = null;
+      spies = {
+        logSpy: spy(console, 'log'),
+        infoSpy: spy(console, 'info'),
+        warnSpy: spy(console, 'warn'),
+        errorSpy: spy(console, 'error')
+      };
+    });
+
+    afterEach(() => {
+      spies.logSpy.restore();
+      spies.infoSpy.restore();
+      spies.warnSpy.restore();
+      spies.errorSpy.restore();
+    });
+
+    it('calls custom callback with correct data for calling instance', () => {
+      client1.info('info message!');
+      expect(result.message).to.equal('info message!');
+      expect(result.args[0]).to.equal('info message!');
+      expect(result.level).to.equal('info');
+      expect(result.type).to.equal('@firebase/test-logger');
+      expect(spies.infoSpy.called).to.be.true;
+      spies.infoSpy.resetHistory();
+      client2.info('another info message!');
+      expect(result.message).to.equal('another info message!');
+      expect(result.args[0]).to.equal('another info message!');
+      expect(result.level).to.equal('info');
+      expect(result.type).to.equal('@firebase/other-logger');
+      expect(spies.infoSpy.called).to.be.true;
+    });
+
+    it('parses multiple arguments correctly', () => {
+      client1.info('info message!', ['hello'], 1, { a: 3 });
+      expect(result.message).to.equal('info message! ["hello"] 1 {"a":3}');
+      expect(result.args).to.deep.equal(['info message!', ['hello'], 1, { a: 3 }]);
+      expect(result.level).to.equal('info');
+      expect(result.type).to.equal('@firebase/test-logger');
+    });
+
+    it('calls custom callback when log call is above set log level', () => {
```

Comment:
Can you please add a comment that log level is defaulted to`INFO`  here and in other tests where we assume the default log level?

Reply:
Done. It is also in this `describe` block title.